server/shared/mail/templates/comment.mjml

```diff
@@ -3,33 +3,52 @@
   <mj-body padding="10px">
     <mj-include path="./components/header.mjml"/>
     <mj-section
-      padding="0 10px"
+      padding="0 10px 20px"
       text-align="left">
-      <mj-column border-bottom="1px solid #ccc">
+      <mj-column>
         <mj-text
           mj-class="content"
           padding-left="0"
           color="#444"
           font-size="16px">
-          <b>{{author.label}}</b> left a comment on {{repositoryName}}, {{topic}}.
-        </mj-text>
-      </mj-column>
-    </mj-section>
-    <mj-section padding="20px 10px">
-      <mj-column
-        background-color="#f1f1f1"
-        border-radius="8px">
-        <mj-text
-          mj-class="content"
-          padding="20px"
-          color="#444"
-          font-size="16px">
-          {{content}}
+          <b>{{author.label}}</b> left a comment on <b>{{repositoryName}}</b>, <b>{{topic}}</b>.
         </mj-text>
       </mj-column>
     </mj-section>
     <mj-section padding="0 10px">
-      <mj-column border-top="1px solid #ccc">
+			<mj-column border-bottom="2px dashed #d2d2d2">
+        <mj-table padding="10px">
+          <tr class="content" style="display: flex;">
+            <td>
+              <img class="avatar" src="{{author.imgUrl}}" />
+            </td>
+            <td style="padding: 0 0 0 20px">
+              <div class="label">{{author.label}}</div>
+              <div>{{content}}</div>
+            </td>
+          </tr>
+        </mj-table>
+       </mj-column>
+    </mj-section>
+    {{#previousComments}}
```

Comment:
Consider extracting into separate component.

Reply:
I don't think it is possible due to problems described in the render function. If we were to import separate component here, `mustache` render would not catch that component and `mjml2html ` would mess up mustache tags, second mustache render would not be able to render that component properly.